app/routes/events/components/EventEditor/EditorSection/Details.tsx

```diff
@@ -0,0 +1,166 @@
+import { Flex } from '@webkom/lego-bricks';
+import cx from 'classnames';
+import { Field } from 'react-final-form';
+import {
+  SelectInput,
+  TextInput,
+  CheckBox,
+  DatePicker,
+} from 'app/components/Form';
+import fieldStyles from 'app/components/Form/Field.css';
+import MazemapLink from 'app/components/MazemapEmbed/MazemapLink';
+import { EventTypeConfig } from 'app/routes/events/utils';
+import styles from '../EventEditor.css';
+import type { EditingEvent } from 'app/routes/events/utils';
+
+type Props = {
+  values: EditingEvent;
+};
+
+const Details: React.FC<Props> = ({ values }) => {
+  return (
+    <>
+      <Flex gap={10}>
```

Comment:
Might not look good here since I haven't checked, but ideally we should use more "standard" spacing like e.g. `0.5rem` or `1rem`. This applies to the other spaces below

Reply:
In that aspect, maybe we should consider adding variables such as `--padding-sm`, etc. Or is that a bit over the top?

Reply:
Yes! I've wanted to do that for a long time - just haven't gotten around to it since everything is so random atm